src/pages/AccessDeniedOuter.tsx

```
@@ -0,0 +1,149 @@
+    }
+    50% {
+      transform: scale(1.1);
+      opacity: 0.8;
+    }
+    100% {
+      transform: scale(1);
+      opacity: 1;
+    }
+  `;
+
+  const handleLogout = async () => {
```

ESLint

[error] 'handleLogout' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)